src/copick_utils/writers/write.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, Dict, List
+import ome_zarr.writer
+import zarr
+
+def tomogram(
+    run,
+    inputVol,
+    voxelSize=10,
+    tomo_algorithm="wbp"
+    ):
+    """
+    Write a OME-Zarr segmentation into a Copick Directory.
+
+    Parameters:
+    - run: The run object, which provides a method to create a new segmentation.
+    - segmentation: The segmentation data to be written.
+    - voxelsize (float): The size of the voxels. Default is 10.
+    """
+
+    # Create a new segmentation or Read Previous Segmentation
+    vs = run.get_voxel_spacing(voxelSize)
+    if vs is None:
+        vs = run.new_voxel_spacing(
+            voxel_size=voxelSize,
+        )
+        tomogram = vs.new_tomogram(tomo_algorithm)
+    else:
+        tomogram = vs.get_tomogram(tomo_algorithm)
+        
+    # Write the zarr file
+    loc = tomogram.zarr()
+    root_group = zarr.group(loc, overwrite=True)
+
+    ome_zarr.writer.write_multiscale(
+        [inputVol],
+        group=root_group,
+        axes=ome_zarr_axes(),
+        coordinate_transformations=[ome_zarr_transforms(voxelSize)],
+        storage_options=dict(chunks=(256, 256, 256), overwrite=True),
+        compute=True,
+    )
+
+def segmentation(
+    run,
+    inputSegmentVol,
+    userID,
+    segmentationName="segmentation",
+    sessionID="0",
+    voxelSize=10,
+    multilabel_seg = True
+    ):
+    """
+    Write a OME-Zarr segmentation into a Copick Directory.
+
+    Parameters:
+    - run: The run object, which provides a method to create a new segmentation.
+    - segmentation: The segmentation data to be written.
+    - voxelsize (float): The size of the voxels. Default is 10.
+    """
+
+    # Create a new segmentation or Read Previous Segmentation
+    seg = run.get_segmentations(name=segmentationName, user_id=userID, session_id=sessionID)
+
+    if len(seg) == 0 or seg[0].voxel_size != voxelSize:
+        seg = run.new_segmentation(
+            voxel_size=voxelSize,
+            name=segmentationName,
+            session_id=sessionID,
+            is_multilabel=multilabel_seg,
+            user_id=userID,
+        )
+    else:
+        seg = seg[0]
+
+
+    # Write the zarr file
+    loc = seg.zarr()
+    root_group = zarr.group(loc, overwrite=True)
+
+    ome_zarr.writer.write_multiscale(
```

Comment:
Same for segmentations: https://github.com/copick/copick/blob/90ec8a25e201a7439e70ef24d1642e244bc01e76/src/copick/models.py#L275